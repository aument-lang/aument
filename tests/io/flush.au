file = io::open("text.txt", "r");
io::flush(file);