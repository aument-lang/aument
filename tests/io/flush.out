fopen text.txt, rb
fflush
